cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -179,6 +179,10 @@ spec:
         # The duration between discovering devices in the rook-discover daemonset.
         - name: ROOK_DISCOVER_DEVICES_INTERVAL
           value: "60m"
+        # Rook-discover daemonset will use "ceph-volume inventory" command to extend
+        # information about storage devices in each node.
+        - name: ROOK_DISCOVER_USES_CEPH_VOLUME
```

Comment:
The ceph operator should always call ceph-volume, right? So I don't think we need to expose this variable in the operator.yaml

Reply:
I thought that command parameters also were set as env-vars. (i didn't understand why) ... Better in this way. Removed